server/apps/main/templates/main/partials/footer.html

```diff
@@ -13,21 +13,21 @@
     <div class="col-lg-3 footer-text">
       <h5>Allied Organisations</h5>
       <p>Autistica</p>
-      <p>National Autistic Society</p>
-      <p>Samaritans</p>
+      <p>The Alan Turing Institute</p>
+      <p>Open Humans</p>
     </div>
 
     <div class="col-lg-3 footer-text">
       <h5>Policies</h5>
       <p>Ethics</p>
       <p>Terms of Use</p>
-      <p>Code of Conduct</p>
+      <p> <a href="{% url 'main:code_of_conduct' %}">Code of Conduct</a></p>
     </div>
 
     <div class="col-lg-3 footer-text">
     <h5>About Us</h5>
-      <p>GitHub</p>
-      <p>Glossary</p>
-      <p>Publications</p>
+      <p>Platform Usage</p>
+      <p>Explore AutSPACEs</p>
+      <p>Data Usage</p>
```

Comment:
I think I would at least leave (and actually add) the link to GitHub here. And I'm not sure which links we should add otherwise, maybe worth having a larger think about.

Reply:
I've included AutSPACEs GitHub and added in a link